docs/website/src/pages/index.tsx

```diff
@@ -0,0 +1,135 @@
+                  src="https://ghbtns.com/github-btn.html?user=redhat-developer&amp;repo=odo&amp;type=star&amp;count=true&amp;size=large"
+                  width={160}
+                  height={30}
+                  title="GitHub Stars"
+                  frameBorder={0}
+                />
+              </span>
+            </div>
+          </div>
+        </div>
+        <div className={clsx(styles.title, styles.titleDark)}>
+          <div className={styles.titleInner}>
+            Visit our <Link to="/blog"> our blog </Link>for the newest developments on odo
+
+          </div>
+        </div>
+        <div className={styles.overview}>
+          <div className="container text--center margin-top--md">
+            <div className="row">
+              <div className="col col--5 col--offset-1">
+                    <video className={styles.loopVideo} autoPlay loop muted><source src="/video/container_ship.webm" type="video/webm"/></video>
+                <h2 className={clsx(styles.featureHeading)}>
+                  Use on <b className={styles.kubernetesFont}>Kubernetes</b> or <b className={styles.openshiftFont}>OpenShift</b>
+                </h2>
+                <p className="padding-horiz--md">
+    We provide first class support for both Kubernetes and OpenShift. Choose your favourite orchestrator and deploy your application.
+                </p>
+              </div>
+              <div className="col col--5">
+                    <video className={styles.loopVideo} autoPlay loop muted><source src="/video/coding.webm" type="video/webm"/></video>
+                <h2 className={clsx(styles.featureHeading)}>
+                  Deploy fast and often
+                </h2>
+                <p className="padding-horiz--md">
+    Spend less time maintaining your deployment infrastructure and more time coding. Immediately have your application running each time you compile.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className={clsx(styles.overview, styles.overviewAlt)}>
+          <div className="container text--center margin-top--lg">
+            <div className="row">
+              <div className="col">
+                <h2 className={clsx(styles.featureHeading)}>
+    Standalone client
+                </h2>
+                <p className="padding-horiz--md">
+    <code>odo</code> is a standalone tool that communicates directly with the container orchestrator API. There is no requirement for a daemon or server process.
+                </p>
+              </div>
+              <div className="col">
+                <h2 className={clsx(styles.featureHeading)}>
+                  Built for container engineers
+                </h2>
+                <p className="padding-horiz--md">
+      Built from the ground up with application development on container orchestrators in mind. Each command has been carefuly crafted for application container development.
```

Comment:
OK, I'm starting to feel that we should actually use "Kubernetes" because other container orchestrators like Hashicorp Nomad or Apache Mesos won't be a good place to try odo.
```suggestion
      Built from the ground up with application development on Kubernetes in mind. Each command has been carefuly crafted for application container development.
```